sdks/python/apache_beam/runners/portability/universal_local_runner.py

```diff
@@ -50,42 +37,30 @@
 
 
 class UniversalLocalRunner(runner.PipelineRunner):
-  """A BeamRunner that executes Python pipelines via the Beam Job API.
+  """
+    Experimental: No backward compatibility guaranteed.
+    A BeamRunner that executes Python pipelines via the Beam Job API.
 
-  By default, this runner executes in process but still uses GRPC to communicate
-  pipeline and worker state.  It can also be configured to use inline calls
-  rather than GRPC (for speed) or launch completely separate subprocesses for
-  the runner and worker(s).
+    This runner is a stub and does not run the actual job.
+    This runner schedules the job on a job service. The responsibility of
+    running and managing the job lies with the job service used.
   """
 
-  def __init__(
-      self,
-      use_grpc=True,
-      use_subprocesses=False,
-      runner_api_address=None,
-      docker_image=None):
-    if use_subprocesses and not use_grpc:
-      raise ValueError("GRPC must be used with subprocesses")
+  # TODO(angoenka): Read all init parameters from pipeline_options.
+  def __init__(self,
+               runner_api_address=None,
+               job_service_address=None,
```

Comment:
Do we need runner_api_address and docker_image?

Reply:
Not sure, I need to check with @robertwb about the intent of runner_api_address.
docker_image and other parameters in constructor should be moved to pipeline_options as job_service will interpret it.